ophan_calls.py

```diff
@@ -97,3 +98,23 @@ def fetch(self):
             return []
 
         return [(entry['url'], entry['count']) for entry in json.loads(most_viewed_result)]
+
+
+class MostPopularByTagFetcher(object):
+    def __init__(self, ophan_client, keywordTag=None):
+        self.ophan_client = ophan_client
+        self.keywordTag = keywordTag
+
+    def fetch(self):
+        params = {
+            "age" : str(86400),
+            "api-key" : self.ophan_client.api_key
+        }
+
+        path = '/api/mostread/keywordtag/'+ urllib.quote(self.keywordTag, safe='')
+
+        headers, most_viewed_result = self.ophan_client.do_get(path, params=params, add_base=True)
+        if not most_viewed_result:
+            return []
```

Comment:
Can we get away with returning `None`, rather than an empty list?

Makes the code a _little_ simpler:

```python
if most_viewed_result:
    return [(entry['url'], entry['count']) for entry in json.loads(most_viewed_result)]
````